pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -148,6 +148,12 @@ spec:
               value: unix:///csi/csi-addons.sock
             {{ end }}
           imagePullPolicy: {{ .ImagePullPolicy }}
+          {{ if .HostPathRequiresPrivileged }}
+          {{ if .CSILogRotation }}
```

Comment:
Golang templates have an [and](https://pkg.go.dev/text/template#hdr-Functions) function. Would it make more sense to use that here?

Reply:
Yaa I was finding any function like this, thanks

Reply:
Updated @BlaineEXE